ord_schema/proto/reaction.proto

```diff
@@ -479,28 +475,25 @@ message ReactionConditions {
 }
 
 message TemperatureConditions {
-  message TemperatureControl {
-    enum TemperatureControlType {
-      UNSPECIFIED = 0;
-      CUSTOM = 1;
-      AMBIENT = 2;
-      OIL_BATH = 3;
-      WATER_BATH = 4;
-      SAND_BATH = 5;
-      ICE_BATH = 6;
-      DRY_ALUMINUM_PLATE = 7;
-      MICROWAVE = 8;
-      DRY_ICE_BATH = 9;
-      AIR_FAN = 10;
-      LIQUID_NITROGEN = 11;
-    }
-    TemperatureControlType type = 1;
-    // Specify, e.g., ICE_BATH components (e.g., "ice water, salt"),
-    // MICROWAVE power, AIR_FAN speed, etc.
-    string details = 2;
+  enum TemperatureControl {
```

Comment:
I'd keep this as `TemperatureControlType`.

Reply:
Fixed these, though they are causing tests to fail since the original submessage names were `{}Conditions.{}Control`